jscomp/bsb/templates/basic/bsconfig.json

```diff
@@ -4,5 +4,10 @@
   "sources": {
     "dir" : "src",
     "subdirs" : true
-  }
+  },
+  "package-specs": {
+    "module": "es6",
+    "in-source": true
+  },
+  "suffix": ".bs.js"
```

Comment:
Since this is the starter template, we get to decide our convention from here on. Should it be `.js`? `.mjs`? Or stay as `.bs.js`?